lib/Extension/PHPUnit/Tests/Unit/CodeTransform/Domain/Refactor/GenerateTestMethodsTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Phpactor\Extension\PHPUnit\Tests\Unit\CodeTransform\Domain\Refactor;
+
+use Generator;
+use Phpactor\CodeTransform\Domain\SourceCode;
+use Phpactor\CodeTransform\Tests\Adapter\WorseReflection\WorseTestCase;
+use Phpactor\Extension\PHPUnit\CodeTransform\Domain\Refactor\GenerateTestMethods;
+
+class GenerateTestMethodsTest extends WorseTestCase
+{
+    /**
+     * @param array<string> $expected
+     *
+     * @dataProvider dataCanMethodBeGenerated
+     */
+    public function testCanMethodBeGenerated(string $source, array $expected): void {
+        $sourceCode = SourceCode::fromStringAndPath('<?php '.$source, 'file:///source');
+
+        // Adding a phpunit stub
+        $this->workspace()->put('TestCase.php', <<<PHP
+            <?php
+
+            namespace PHPUnit\Framework;
+
+            class TestCase {}
+        PHP);
```

Comment:
Is that the correct way to add the file to the workspace? Because it still doesn't find it.

Reply:
Never mind I forgot the use statement in the other code. :D